### PR TITLE
[AST] to_sql for STATEMENTS 

### LINKED ISSUE #822: [AST] Implement to_sql func for STATEMENTS
## Description
implement to_sql functions for ast STATEMENTS
- currently, to_sql functions for `ShowColumns`, `DropTable`, `CreateIndex`, `DropIndex`, `StartTransaction`, `Commit`, `Rollback`, `ShowVariable`, `ShowIndexes` (and `SELECT`, `VALUES`) are unimplemented.

### PR DESCRIPTION
**Pull request by ding-young**
## Issue
resolves #822

## Description
- to_sql for `create index` is incomplete since to_sql for `orderByExpr` is unimplemented 
- further task : impl SELECT, VALUES

### CODE REVIEW

File: core/src/ast/mod.rs [hunk 1]
```diff
@@ -203,6 +206,48 @@ impl ToSql for Statement {
...
+                match if_exists {
+                    true => format!("DROP TABLE IF EXISTS {}", names),
+                    false => format!("DROP TABLE {}", names),
+                }
+            }
+            #[cfg(feature = "index")]
+            Statement::CreateIndex {
+                name,
+                table_name,
+                column: _column,
```
Comment (ever0de) [inline core/src/ast/mod.rs:224]: [nit] If not used, the field can be omitted.
```suggestion
                ..
```

File: core/src/ast/mod.rs [hunk 2]
```diff
@@ -132,7 +132,7 @@ impl ToSql for Statement {
     fn to_sql(&self) -> String {
         match self {
             Statement::ShowColumns { table_name } => {
-                format!("SHOW COLUMNS from {}", table_name.to_sql())
+                format!("SHOW COLUMNS FROM {}", table_name.to_sql())
```
Comment (panarch) [inline core/src/ast/mod.rs:135]: 👍

File: core/src/ast/mod.rs [hunk 3]
```diff
@@ -485,6 +547,99 @@ mod tests {
...
+                    ObjectName(vec!["Foo".to_string()]),
+                    ObjectName(vec!["Bar".to_string()])
+                ]
+            }
+            .to_sql()
+        );
+    }
+
+    #[test]
+    #[cfg(feature = "index")]
```
Comment (ever0de) [inline core/src/ast/mod.rs:584]: Good :+1:

**Review by ever0de — APPROVED** [review]
I left only simple improvements.

Looks good 👍

File: core/src/ast/mod.rs [hunk 4]
```diff
@@ -203,6 +206,48 @@ impl ToSql for Statement {
...
+                }
+            }
+            #[cfg(feature = "index")]
+            Statement::CreateIndex {
+                name,
+                table_name,
+                column: _column,
+            } => {
+                format!(
+                    "CREATE INDEX {} ON {} (..order_by_expr..)",
```
Comment (ever0de) [inline core/src/ast/mod.rs:227]: [context]: <https://github.com/gluesql/gluesql/pull/812> must be completed to enter. TODO comments will be added.

File: core/src/ast/mod.rs (hunk 1 above)
Comment (panarch) [reply core/src/ast/mod.rs:224]: In this case, it would be good to specify `column` explicitly for reminder which we will handle this field 'column' soon

**Review by panarch — APPROVED** [review]

**Comment by panarch** [comment]
Looks all good! Thanks a lot 👍

**Pull request merged**